airgun/session.py

```diff
@@ -301,10 +301,8 @@ def take_screenshot(self):
         session happens.
         """
         now = datetime.now()
-        path = os.path.join(
-            settings.selenium.screenshots_path,
-            now.strftime('%Y-%m-%d'),
-        )
+        path = settings.selenium.screenshots_path
+
```

Comment:
@dosas Thanks for creating the PR, I appreciate the work you added. Looking at the function doc string this was added long back, looking at this statement

 `All directories will be created if they don't exist. Make sure that the user running Robottelo has the right permissions to create files and directories.` this could be the reason this was added. I see some value in that. 

- It dynamically creates directories based on the current date, ensuring that the screenshots or files are organized in a structured manner. Imagine if someone specifies the '/tmp' path here; all the screenshots would be jumbled together in a single directory. It's much better to have them organized by date for better isolation.
- 

@jyejare, @ogajduse what are your thoughts?

Reply:
There is an easy fix for that don't use `/tmp` but use `/tmp/screenshots`.
Don't take away all responsibility from the user.

Reply:
> I would even go as far and not put the date information into the filename, since this is duplicate information that is already provided by the filesystem.

I have a different perspective on this. If we don't include the date and time in the screenshot filenames using (DateTime) f'{self.name}-screenshot-YYYY-mm-dd_HH_MM_SS.png', there's a risk of overwriting the same screenshots when running the test again.

Don't fix something that isn't broken.

Reply:
Good point I did not think of that